pkg/bpmn_engine/engine_jobs_activated.go

```diff
@@ -18,16 +19,32 @@ type activatedJob struct {
 	processDefinitionKey     int64
 	elementId                string
 	createdAt                time.Time
+	scope                    variable_scope.VarScope
+	localScope               variable_scope.VarScope
 }
 
 // ActivatedJob represents an abstraction for the activated job
 // don't forget to call Fail or Complete when your task worker job is complete or not.
 type ActivatedJob interface {
-	ProcessInstance
+	// GetInstanceKey get instance key from processInfo
+	GetInstanceKey() int64
```

Comment:
This change does not relate to introducing variable context. Please remove it in this PR and create a separate PR, if required.
Also, this seems to be redundant, since there is already: ```GetProcessInstanceKey()``` available.